runtime/runc_jailer.go

```diff
@@ -46,21 +46,23 @@ type runcJailer struct {
 	runcBinaryPath string
 	uid            uint32
 	gid            uint32
+	vmID           string
 }
 
 const firecrackerFileName = "firecracker"
 
-func newRuncJailer(ctx context.Context, logger *logrus.Entry, ociBundlePath, runcBinPath string, uid, gid uint32) (*runcJailer, error) {
+func newRuncJailer(ctx context.Context, logger *logrus.Entry, ociBundlePath string, service *service, uid, gid uint32) (*runcJailer, error) {
```

Comment:
nit: I'd personally prefer we just separately provide `runcBinaryPath` and `vmID` given that `service` is a pretty massive object, it just makes it easier to know+control what the runc jailer is actually depending on

Reply:
Fixed